src/internal.c

```diff
@@ -4984,8 +5014,10 @@ int EccVerify(WOLFSSL* ssl, const byte* in, word32 inSz, const byte* out,
         ret = ssl->ctx->EccVerifyCb(ssl, in, inSz, out, outSz, keyBuf, keySz,
             &ssl->eccVerifyRes, ctx);
     }
-    #if !defined(WOLFSSL_RENESAS_SCEPROTECT) && \
-        !defined(WOLFSSL_RENESAS_TSIP_TLS)
+    #if defined(HAVE_PK_CALLBACKS) && defined(WOLFSSL_MAXQ108x)
+    if ((ret == NOT_COMPILED_IN) || ssl->options.side != WOLFSSL_CLIENT_END)
```

Comment:
What is this? Fallback to software if not client  or it returns NOT_COMPILED_IN? Why only client?

Reply:
> Fallback to software if it returns NOT_COMPILED_IN?

Yes. I think this is a rather nice.  When wolfssl calls a callback, wolfssl doesn't know whether or not the callback actually supports the operation.  For example, an operation might support certain curves (P256) but not other ones (P521) so if it  returns `NOT_COMPILED_IN` then that signals a fallback to wolfssl's implementation.  Does this seem reasonable?

> Why only client?

Ah this is a left over artifact from the way the customer gave us the patch.  The MAXQ hardware does not support being the server; only being the client. So they only want the maxq10xx APIs called if this is the client side. I've had discussions with the customer and we came up with a better approach where when support for MAXQ is turned on at configure time, then support for being the server side of a TLS connection will be disabled.  When I make that change I will also remove all these checks for `WOLFSSL_CLIENT_END`.  That will likely come after I've done all the possible callback conversions.

Reply:
"Only client" fixed. Keeping fallback.